lib/parseOptions.js

```diff
@@ -47,6 +50,12 @@ export default function parseOptions (params) {
     throw new Error('Please provide the proxy config in the following format:\nhost:port or user:password@host:port')
   }
 
+  options.startTime = moment()
+
+  if (!options.errorLogFile) {
```

Comment:
you need ` resolve(process.cwd(), options.errorLogFile)` also if it is set, because user may pass in `mylogfile.json` fro example